scripts/safari.js

```diff
@@ -7051,10 +7120,36 @@ function Safari() {
             winner = this.team1;
             safaribot.sendHtmlAll("The " + toColor(this.team1Name, this.team1Color) + " (" + readable(winner, "and") + ") has won the Faction War!", safchan);
         }
+        
+        var mvp = [], mvpPoints = 0, e, i, teams = this.playerTeams, id, player, score, mon, totalPoints;
+        
+        for (e in teams) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.